src/api/endpoints/redemption_requests.py

```diff
@@ -27,9 +45,9 @@ def post(cls):
                 status="fail"
             ), 400)
 
-        name = payload.get("name")
+        name = payload.get("reason")
         value = payload.get("value")
-        country_input = payload.get("country")
+        country_input = payload.get("center")
 
         country = Country.query.filter_by(name=country_input).first()
```

Comment:
Shouldn't this be referencing Center? The work on Countries being renamed was merged.